Baby Monitor/Source Files/Services/Websocket/WebSocketEventMessageService.swift

```diff
@@ -48,6 +56,9 @@ final class WebSocketEventMessageService: WebSocketEventMessageServiceProtocol {
             if let webRtcSdpErrorMessage = event.webRtcSdpErrorMessage {
                 self?.remoteStreamConnectingErrorPublisher.onNext(webRtcSdpErrorMessage)
             }
+            if let confirmationID = event.confirmationId {
+                self?.remoteConfimationIdPublisher.onNext(confirmationID)
```

Comment:
Please unwrap `self` in the guard above to avoid checking all those `if`'s when `self == nil`